app/models/post.rb

```diff
@@ -0,0 +1,5 @@
+class Post < ApplicationRecord
```

Comment:
- [ ] As part of this module requirement, _Post_ model should have a method that updates the **posts counter** for a user. This should be similar to:
```ruby
# app/models/post.rb
...

def update_post_counter
  author.increment!(:posts_counter)
end
```